config/default/layout_builder_styles.style.block_padding_all.yml

```diff
@@ -5,7 +5,7 @@ dependencies: {  }
 _core:
   default_config_hash: 3YxL6BonqkTWB6h5CSCKqjKEZDXjD_mQmYYLylDCMxY
 id: block_padding_all
-label: 'All sides'
+label: 'All sides (default)'
```

Comment:
I think we may want to set this via code. There are examples of this in `layout_builder_custom.module`.

Reply:
Does this still need to be moved to layout_builder_custom.module?

Reply:
I think adjusting the label to just `Default` allows us to bypass that?

Reply:
yeah sorry. didn't connect the remaining conversation.